src/main/java/hw_jdi/pages/MetalsAndColorsPage.java

```diff
@@ -0,0 +1,60 @@
+package hw_jdi.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import hw_jdi.entities.MetalsAndColorsData;
+
+import java.util.List;
+
+@Url("/metals-colors.html")
+public class MetalsAndColorsPage {
```

Comment:
It will be better to create a Form instead of defining all elements on page directly.